syntax = "proto2";

package eflect.data;

message ProcStatSample {
  required int64 timestamp = 1;
  required int64 cpu = 2;
  required int64 user = 3;
  required int64 nice = 4;
  required int64 system = 5;
  required int64 idle = 6;
  required int64 iowait = 7;
  required int64 irq = 8;
  required int64 softirq = 9;
  required int64 steal = 10;
  required int64 guest = 11;
  required int64 guest_nice = 12;
}
